### DISCUSSION TITLE
Entities today will stop working after midnight

### DISCUSSION THREAD

**Discussion by sitvesep** [mletenay/home-assistant-goodwe-inverter#279] [General]
The last two updates are causing me to get an error as soon as it is midnight, when the entity states are supposed to be reset: **the entity is not numeric**.  Only entities **today** will stop working.  If I revert to  v0.9.9.17, everything is fine.

![battery](https://github.com/mletenay/home-assistant-goodwe-inverter/assets/136500212/f097b7ce-622b-460d-90c8-183bec4199fc)
![PV](https://github.com/mletenay/home-assistant-goodwe-inverter/assets/136500212/9df9420e-933c-4c2d-b9f8-bc94d0abf8ac)

This problem I was also noticed by some people on the FB group about PV. 

Does anyone also have experience with this error ?

GW10K-ET
Firmaware: 04029-11-S11/02041-27-S00
 

Thank you

**Comment by janchlebek** [comment]
I am using v0.9.9.18 and do not see this behaviour, everything is working as expected (GW10K-ET, FW 1127).

**Comment by mletenay** [comment]
Aaaaaargh !
I fixed one thing and broke another.
This most likely caused by fix of this - https://github.com/home-assistant/core/issues/116762

**Comment by mletenay** [comment]
@sitvesep Where precisely do you see these warnings ?
Since I just see Unavailable entities after midnight, which I start to think is actually a proper state and I could not find not-a-number error anywhere ...

**Reply by sitvesep** [reply]
I will send the log in the evening by appointment on Discort ( I have a different name than here)

**Comment by Legnava89** [comment]
In my opinion, this is not the right state. On an oled or lcd display it will throw a NaN (unknown) error The correct state is 0.
Many people would like to go back to state 0 like version v0.9.9.17

**Comment by mletenay** [comment]
Fixed in v0.9.9.20
